src/style.css

```diff
@@ -0,0 +1,136 @@
+@import "./variables.css";
+
+body {
+  margin: 0;
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-style: normal;
+  /* #region variables */
+  --card-width: 200px;
+  --black-color: #060b35;
+  --grey-color: #616070;
+  --white-color: #fff;
+  --blue-color: #00acdc;
+  --border-color-main: #f3f3f3;
+  /* #endregion */
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: var(--card-width);
```

Comment:
width is external style as well
<img width="1010" alt="image" src="https://user-images.githubusercontent.com/102411130/208073477-c1cbad8e-64ec-4177-ad78-371a8ad3e858.png">

Reply:
Set the width of the entire body tag to not use the mix. Would that be acceptable?

Reply:
Is it correct to use a modifier in this case?